test/off_chain_debts.js

```diff
@@ -0,0 +1,61 @@
+const Reverter = require('./helpers/reverter');
+const Asserts = require('./helpers/asserts');
+const OffChainDebts = artifacts.require('./OffChainDebts.sol');
+
+contract('OffChainDebts', function(accounts) {
+  const reverter = new Reverter(web3);
+  afterEach('revert', reverter.revert);
+
+  const asserts = Asserts(assert);
+  const OWNER = accounts[0];
+  let debts;
+
+  before('setup', () => {
+    return OffChainDebts.deployed()
+    .then(instance => debts = instance)
+    .then(reverter.snapshot);
+  });
+
+  it('should allow to repay', () => async function () {
+    const borrower = accounts[3];
+    const value = 1000;
+    await debts.borrow.call(value, {from: borrower});
+    await debts.repay.call(borrower, value, {from: borrower});
```

Comment:
you have nothing to repay because when you use .call you just simulate the function.
this test will never fail.